src/webview/create-component/createComponentLoader.ts

```diff
@@ -82,10 +92,13 @@ export default class CreateComponentLoader {
              * The panel requested the list of workspace folders. Respond with this list.
              */
             case 'getWorkspaceFolders': {
-                void CreateComponentLoader.panel.webview.postMessage({
-                    action: 'workspaceFolders',
-                    data: vscode.workspace.workspaceFolders,
-                });
+                if (vscode.workspace.workspaceFolders !== undefined) {
+                    const workspaceFolderUris: Uri[] = vscode.workspace.workspaceFolders.map(wsFolder => wsFolder.uri);
```

Comment:
it would be nice to filter out folders that already have a devfile and show a popup when the user adds a folder that already has a devfile.